src/database.ts

```diff
@@ -1989,12 +1997,27 @@ class Database extends common.GrpcServiceObject {
    * });
    * ```
    */
-  getTransaction(): Promise<[Transaction]>;
-  getTransaction(callback: GetTransactionCallback): void;
+  getTransaction(callback: GetTransactionCallback): Promise<[Transaction]>;
```

Comment:
(I'm not super familiar with how `promisify` works, so it could be that I'm overly worried here) This overload looks a bit strange. It takes both a callback as an argument, __and__ returns a promise. Normally it is either accepting a callback __or__ returning a promise.

Reply:
Updated